app/build.gradle

```diff
@@ -14,7 +14,7 @@ android {
         targetSdkVersion 29
         versionCode 15
         versionName "2.0.2"
-        ndkVersion "20.0.5594570"
+        ndkVersion "21.0.6113669"
```

Comment:
This will break Buildozer, because it only has one NDK version available. Not sure if you are OK with that.

Reply:
This 21.0.6113669 is also what is used in ics-openvpn, so I just used the same version here... or did I make a typo?

Reply:
I think Buildozer looks at the one on the top level. We can try, maybe you are right :)

Reply:
102: > No version of NDK matched the requested version 21.0.6113669. Versions available locally: 20.0.5594570, 21.1.6352462

Aha... but ics-openvpn uses 21.0.6113669, so I'd either have to also modify that, or hmm?